libsolidity/ast/Types.cpp

```diff
@@ -949,11 +949,22 @@ bool RationalNumberType::operator==(Type const& _other) const
 	return m_value == other.m_value;
 }
 
+string numberToString(dev::bigint num)
+{
+	string str = num.str();
+	if (str.size() > 32)
```

Comment:
Should we support 256 bit values unaltered? That would be 78 (?) digits.

Reply:
Unless you are copy-pasting them, you are probably only verifying the first/last digits. I think long numbers just add noise to the error message.

Reply:
I think I'm in favour on your argument now.